src/sum_of_nth_terms.py

```diff
@@ -0,0 +1,15 @@
+"""A function that returns the sum of a series up to the nth value.
```

Comment:
Nice job adding a docstring to the file.  It is giving my linter an error though, so see if you can get it to pass.  If your linter isn't showing this as an error, you might want to get a new linter.  Let me know if you need any help with this!